src/dashboard/src/templates/main/tasks.html

```diff
@@ -11,21 +12,55 @@
       {% for item in objects %}
         <tr class="{% cycle 'odd' 'even' %}{% ifnotequal item.exitcode 0 %} exiterror{% endifnotequal %}">
           <td>
-            <span>Task UUID:</span> {{ item.taskuuid }}<br />
-            <span>File UUID:</span> {{ item.fileuuid }}<br />
-            <span>File name:</span> {{ item.filename }}<br />
-            <span>Client:</span> {{ item.client }}<br />
+            {% blocktrans with uuid=item.taskuuid %}
+              <span>Task UUID:</span> {{ uuid }}
+            {% endblocktrans %}
+            <br />
+
+            {% blocktrans with uuid=item.fileuuid %}
+              <span>File UUID:</span> {{ uuid }}
+            {% endblocktrans %}
+            <br />
+
+            {% blocktrans with filename=item.filename %}
+              <span>File name:</span> {{ filename }}
+            {% endblocktrans %}
+            <br />
+
+            {% blocktrans with client=item.client %}
+              <span>Client:</span> {{ client }}
+            {% endblocktrans %}
+            <br />
+
             <span>&nbsp;</span> {{ item.exec_field }} (exit code: {{ item.exitcode }})
           </td>
           <td>
-            <span>Start time:</span> <span class='datetime'>{{ item.starttime.isoformat }}</span><br />
-            <span>End time:</span> <span class='datetime'>{{ item.endtime.isoformat }}</span><br />
-            <span>Created time:</span> <span class='datetime'>{{ item.createdtime.isoformat }}</span><br />
-            <span>Duration:</span> {{ item.duration }} second(s)
+            {% blocktrans with datetime=item.starttime.isoformat %}
+              <span>Start time:</span> <span class="datetime">{{ datetime }}</span>
```

Comment:
Does the HTML & value need to be included in the translation or just the label?  I suppose it might be necessary for RTL reordering.  Somewhat concerned about putting styling in translations.